### DISCUSSION TITLE
cost 0.03$, about what ?

### DISCUSSION THREAD

**Discussion by SheikhElMoctarG** [aws/aws-cli#8462] [Q&A] [answered]
hi, I'm using ec2 in free trial, why it tells cost 0.03$

![image](https://github.com/aws/aws-cli/assets/80722891/776c91d7-0644-4bdb-82cc-d749b8c8f3b5)

why that ?

**Comment by prnr** [comment, marked as answer]
https://repost.aws/knowledge-center/kms-key-charges
Even if you use free tier, you will be charged as much as the amount of KMS service you used.

https://docs.aws.amazon.com/kms/latest/developerguide/deleting-keys.html
I don't really recommend it, but deleting a key in use can stop you from incurring a fee.
